template/component-feedback-widget.html.jinja

```diff
@@ -1,19 +1,19 @@
 <!-- Feedback Widget -->
-<!-- <script src="{{currentpage.prefix}}assets/vendor/feedback-widget.js"></script> -->
-<!-- <div id="feedback-content"></div> -->
+<script src="../../assets/vendor/feedback-widget.js"></script>
```

Comment:
This change (replacing `{{currentpage.prefix}}` with `../../`) is why the feedback widget script is not loading as expected. If you revert this part, it should be fine.